genre.rb

```diff
@@ -12,4 +12,12 @@ def add_item(item)
     @items << item
     item.add_genre(self)
   end
+
+  def to_json(*_arg)
+    {
+      id: @id,
+      name: @name,
+      items: @items.map(&:to_json)
+    }.to_json
+  end
```

Comment:
This looks great my friend, however, I already implemented these functions in my commit, can you please remove it?